app/views/play/level/PlayLevelView.coffee

```diff
@@ -691,6 +698,17 @@ module.exports = class PlayLevelView extends RootView
     $.ajax '/file', type: 'POST', data: body, success: (e) ->
       contactModal.updateScreenshot?()
 
+  onSurfaceContextMenu: (e) ->
+    return unless navigator.clipboard or document.queryCommandSupported('copy')
+    e?.preventDefault?()
+    if @surface.showCoordinates 
```

Comment:
Would we also be able to check for showCoordinates before preventing default behavior?

Currently on early levels that don't have coordinates the "mouse screen pos doesn't exist" log occurs without any context menu on right click.

Reply:
Okay, I can add the showCoordinates check to the return unless statement